Cypress/cypress/integration/Parallel_Tests/Create_Workflow/customWorkflow.spec.js

```diff
@@ -327,6 +329,9 @@ describe("Testing the validation of the final verdict with an existing target ap
 		cy.get("[data-cy=FinishModal]").should("be.visible");
 		cy.get("[data-cy=WorkflowName]").then(($name) => {
 			workflowName = $name.text();
+			cy.get('@workflowNamespace').then((workflowNamespace) => {
+				cy.validateWorkflowExistence(workflowName, workflowNamespace);
+			});
 			return;
 		});
 		cy.get("[data-cy=WorkflowSubject]").then(($subject) => {
```

Comment:
I just saw this, we should be storing these values (Namespace, Workflowname & subject ) before clicking on finish button. In the next it(), we can use those values for validating workflow existance/status after scheduling.